internal/kafka/internal/services/cloud_providers_test.go

```diff
@@ -7,9 +7,9 @@ import (
 
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/clusters"
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/clusters/types"
-
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/db"
+	Error "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
```

Comment:
Is there some reason we are aliasing the `github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors` to `Error`?

Reply:
Yes, the import was set when I generated the unit tests and was conflicting with the other `errors` import that is used elsewhere in the file.

Reply:
Is it outdated? currently I see only one package that ends in "errors"

Reply:
Sorry, I'm not too sure what you mean

Reply:
Looking at the file in the PR it seems that the `Error` import alias is not needed, as it is not conflicting with any other import names

Reply:
When I remove the alias it gives me the error saying that `errors` is re declared [here](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/481ac2577ba1c2a6153e96e146c95e2fcffcf3a1/internal/kafka/internal/services/cloud_providers_test.go#L4). Removing this import creates further errors in the file with some of the other test.

Reply:
aaah sorry I hadn't seen that line. This is correct then :+1:

Reply:
No problem I apprentice the help either way :)